cogs/messages.py

```diff
@@ -31,7 +31,7 @@ async def do_count_per_user(ctx, users_to_process, amount_to_show=10):
 
             i = 0
             for k, v in sorted(counts.items(), key=lambda p: p[1], reverse=True):
-                description += f"`{k} - {v}`\n"
+                description += f"{i+1}. <@{k.id}> - {v}\n"
```

Comment:
You can just do it like this instead...
```suggestion
                description += f"{i+1}. {k.mention} - {v}\n"
```